src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -497,6 +497,8 @@ private int findSaplingSlot()
         return -1;
     }
 
+    //todo: we need to use a different way to get Metadata
+    @SuppressWarnings ("deprecation")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1309)